test/e2e/cloud/gcp/ha_master.go

```diff
@@ -117,23 +117,23 @@ func removeZoneFromZones(zones []string, zone string) []string {
 	return zones
 }
 
-// generateMasterRegexp returns a regex for matching master node name.
-func generateMasterRegexp(prefix string) string {
+// generateControlPlaneRegexp returns a regex for matching master node name.
+func generateControlPlaneRegexp(prefix string) string {
 	return prefix + "(-...)?"
 }
 
-// waitForMasters waits until the cluster has the desired number of ready masters in it.
-func waitForMasters(masterPrefix string, c clientset.Interface, size int, timeout time.Duration) error {
+// waitForControlPlanes waits until the cluster has the desired number of ready masters in it.
+func waitForControlPlanes(masterPrefix string, c clientset.Interface, size int, timeout time.Duration) error {
 	for start := time.Now(); time.Since(start) < timeout; time.Sleep(20 * time.Second) {
 		nodes, err := c.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
 		if err != nil {
 			framework.Logf("Failed to list nodes: %v", err)
 			continue
 		}
 
-		// Filter out nodes that are not master replicas
+		// Filter out nodes that are not controlplane replicas
 		e2enode.Filter(nodes, func(node v1.Node) bool {
-			res, err := regexp.Match(generateMasterRegexp(masterPrefix), ([]byte)(node.Name))
+			res, err := regexp.Match(generateControlPlaneRegexp(masterPrefix), ([]byte)(node.Name))
```

Comment:
i think we should make broader changes, changing variable names like `masterPrefix` as well in the same PR.
[1]